llvm_ir_dataset_utils/compile_time_analysis_tools/combine_outputs.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -o errexit
+
+#Usage:
+#./combine_outputs.sh <language> <storage>
+
+if [ -z "$1" ]; then
+  echo "Missing language argument."
+  exit 1
+else
+  LANGUAGE="$1"
+fi
+
+if [ -z "$2" ]; then
+  echo "Missing storage argument."
+  exit 1
+else
+  STORAGE="$2"
+fi
+
+
+cd ${STORAGE}
+
+mkdir -p ${LANGUAGE}/results
+TARGET_PREFIX="${LANGUAGE}/results/${LANGUAGE}"
+
+boolean=1
+DATA_NAMES=("text_segment" "instruction" "ir_features" "max_pass")
+
+for element in "${DATA_NAMES[@]}"; do
+  if [[ ${element} == ${DATA_NAMES[2]} ]]; then
+    echo "file, BasicBlockCount, BlocksReachedFromConditionalInstruction, Uses, DirectCallsToDefinedFunctions, LoadInstCount, StoreInstCount, MaxLoopDepth, TopLevelLoopCount, TotalInstructionCount, BasicBlocksWithSingleSuccessor, BasicBlocksWithTwoSuccessors, BasicBlocksWithMoreThanTwoSuccessors, BasicBlocksWithSinglePredecessor, BasicBlocksWithTwoPredecessors, BasicBlocksWithMoreThanTwoPredecessors, BigBasicBlocks, MediumBasicBlocks, SmallBasicBlocks, CastInstructionCount, FloatingPointInstructionCount, IntegerInstructionCount, ConstantIntOperandCount, ConstantFPOperandCount, ConstantOperandCount, InstructionOperandCount, BasicBlockOperandCount, GlobalValueOperandCount, InlineAsmOperandCount, ArgumentOperandCount, UnknownOperandCount, CriticalEdgeCount, ControlFlowEdgeCount, UnconditionalBranchCount, IntrinsicCount, DirectCallCount, IndirectCallCount, CallReturnsIntegerCount, CallReturnsFloatCount, CallReturnsPointerCount, CallReturnsVectorIntCount, CallReturnsVectorFloatCount, CallReturnsVectorPointerCount, CallWithManyArgumentsCount, CallWithPointerArgumentCount" \
```

Comment:
Is there a way to get this statically? It would be nice to not have to update this if something in the compiler changes around the function properties analysis.